src/uwtools/tests/utils/test_file.py

```diff
@@ -100,8 +100,10 @@ def test_readable_file(tmp_path):
 
 
 def test_readable_nofile():
-    with file.readable() as f:
-        assert hasattr(f, "read")
+    file._stdinproxy.cache_clear()
+    with patch.object(sys, "stdin", new=StringIO("hello")):
+        with file.readable() as f:
+            assert f.read() == "hello"
```

Comment:
This avoids an occasional error -- for whatever reason only appearing when tests are run in parallel -- where `pytest` will complain about `stdin` being inappropriately used.